README.md

```diff
@@ -41,180 +39,105 @@ In this documentation, several important terms are used and are intended to have
 
-To remove more than one operator, repeat these steps for each operator namespace.
+### Preparing the Kubernetes cluster to run the WebLogic domain
 
-## Need more help?
+Perform these steps to prepare your Kubernetes cluster to run a WebLogic domain:
 
-We have a public Slack channel where you can get in touch with us to ask questions about using the operator.  To join our channel, please [visit this site to get an invitation](https://weblogic-slack-inviter.herokuapp.com/).  The invitation email will include details of how to access our Slack workspace.  After you are logged in, please come to #operator and say, "hello!"
+* Create the domain namespace.  One or more domains can share a namespace.
+* Define RBAC roles for the domain.
+* Create a Kubernetes secret for the Administration Server boot credentials.
+* Optionally, [create a PV & persistent volume claim (PVC)](kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md) which can hold the domain home, logs, and application binaries.
+* [Configure a load balancer](kubernetes/samples/charts/README.md) to manage the domains and ingresses.
 
-# Developer guide
+### Creating and managing the WebLogic domain
 
-Developers interested in this project are encouraged to read the [Developer guide](site/developer.md) to learn how to build the project, run tests, and so on.  The Developer guide also provides details about the structure of the code, coding standards, and the Asynchronous Call facility used in the code to manage calls to the Kubernetes API.
+To create and manage a WebLogic domain in Kubernetes we create a deployment type, the domain custom resource.   The operator always looks at the custom resource and manages the domain deployment to adjust to the definitions in the custom resource.   This custom resource can also be managed using the Kubernetes command-line interface `kubectl`.  
+* (Point to documentation how to edit the domain inputs YAML and how to create the domain custom resource.)
+* Create Ingress controllers if you are using a load balancer that supports them, such as Traefik or Voyager.
 
-Please take a look at our [wish list](https://github.com/oracle/weblogic-kubernetes-operator/wiki/Wish-list) to get an idea of the kind of features we would like to add to the operator.  Maybe you will see something you would like to contribute to!
+### Managing lifecycle operations
 
-# API documentation
+In Operator 2.0 you can perform lifecycle operations to the WebLogic servers, cluster, or domain.
+* Point to the documentation how to manage lifecycle operations.
 
-Documentation for APIs is provided here:
+### Modifying domain configurations
+You can modify the WebLogic domain configuration for both domain in PV and domain in image:
 
-* [Javadoc](https://oracle.github.io/weblogic-kubernetes-operator/apidocs/index.html) for the operator.
```

Comment:
Did you want me to move the javadoc reference only or the references to the REST APIS and the javadoc? I moved them both to the Developer section where they were originally. That left the reference to "The [Kubernetes RBAC definitions](site/rbac.md) required by the operator'" by itself. I removed it because it didn't make much sense standalone. Let me know where I should put it.

Reply:
Done.